shard-operator/src/main/resources/templates/bridge-ingress-broker.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: bridge-fleet-shard-operator # used for filtering of Deployments created by the controller
+    app.kubernetes.io/created-by: bridge-fleet-shard-operator # Specify the operator
+    app.kubernetes.io/component: ingress # used to specify the component
+  ownerReferences: # used for finding which Processor does this Deployment belong to
+    - apiVersion: ""
+      kind: ""
+      name: ""
+      uid: ""
+  annotations:
+    eventing.knative.dev/broker.class: Kafka
+    x-kafka.eventing.knative.dev/external.topic: ""
```

Comment:
With 1.23 (release next week) it will be `kafka.eventing.knative.dev/external.topic` (the `x-` was removed).

We do in 1.23 support the `x-` value as well, but I'd recommend changing that asap.

Options:
* wait to merge and change until 1.23 (and matching knative 1.2 midstream) is released
* merge now, change shortly after the release